### ISSUE TITLE
2025-05-22, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#177]
# Today's Arxiv Papers

(1) [Learning to Reason via Mixture-of-Thought for Logical Reasoning](http://arxiv.org/abs/2505.15817v1), 2025-05-21

(2) [The Atlas of In-Context Learning: How Attention Heads Shape In-Context Retrieval Augmentation](http://arxiv.org/abs/2505.15807v1), 2025-05-21

(3) [Keep Security! Benchmarking Security Policy Preservation in Large Language Model Contexts Against Indirect Attacks in Question Answering](http://arxiv.org/abs/2505.15805v1), 2025-05-21

(4) [STAR-R1: Spacial TrAnsformation Reasoning by Reinforcing Multimodal LLMs](http://arxiv.org/abs/2505.15804v1), 2025-05-21

(5) [VerifyBench: Benchmarking Reference-based Reward Systems for Large Language Models](http://arxiv.org/abs/2505.15801v1), 2025-05-21

(6) [Reverse Engineering Human Preferences with Reinforcement Learning](http://arxiv.org/abs/2505.15795v1), 2025-05-21

(7) [HCRMP: A LLM-Hinted Contextual Reinforcement Learning Framework for Autonomous Driving](http://arxiv.org/abs/2505.15793v1), 2025-05-21

(8) [Long-Form Information Alignment Evaluation Beyond Atomic Facts](http://arxiv.org/abs/2505.15792v1), 2025-05-21

(9) [Large Language Models as Computable Approximations to Solomonoff Induction](http://arxiv.org/abs/2505.15784v1), 2025-05-21

(10) [Soft Thinking: Unlocking the Reasoning Potential of LLMs in Continuous Concept Space](http://arxiv.org/abs/2505.15778v1), 2025-05-21

(11) [ConvSearch-R1: Enhancing Query Reformulation for Conversational Search with Reasoning via Reinforcement Learning](http://arxiv.org/abs/2505.15776v1), 2025-05-21

(12) [Beyond Hard and Soft: Hybrid Context Compression for Balancing Local and Global Information Retention](http://arxiv.org/abs/2505.15774v1), 2025-05-21

(13) [MIKU-PAL: An Automated and Standardized Multi-Modal Method for Speech Paralinguistic and Affect Labeling](http://arxiv.org/abs/2505.15772v1), 2025-05-21

(14) [An Empirical Analysis of Vulnerability Detection Tools for Solidity Smart Contracts Using Line Level Manually Annotated Vulnerabilities](http://arxiv.org/abs/2505.15756v1), 2025-05-21

(15) [Exploring The Visual Feature Space for Multimodal Neural Decoding](http://arxiv.org/abs/2505.15755v1), 2025-05-21

(16) [Scalable Defense against In-the-wild Jailbreaking Attacks with Safety Context Retrieval](http://arxiv.org/abs/2505.15753v1), 2025-05-21

(17) [Multi-modal Integration Analysis of Alzheimer's Disease Using Large Language Models and Knowledge Graphs](http://arxiv.org/abs/2505.15747v1), 2025-05-21

(18) [Evolutionary Computation and Large Language Models: A Survey of Methods, Synergies, and Applications](http://arxiv.org/abs/2505.15741v1), 2025-05-21

(19) [HybridProver: Augmenting Theorem Proving with LLM-Driven Proof Synthesis and Refinement](http://arxiv.org/abs/2505.15740v1), 2025-05-21

(20) [Alignment Under Pressure: The Case for Informed Adversaries When Evaluating LLM Defenses](http://arxiv.org/abs/2505.15738v1), 2025-05-21

(21) [DEBATE, TRAIN, EVOLVE: Self Evolution of Language Model Reasoning](http://arxiv.org/abs/2505.15734v1), 2025-05-21

(22) [VocalBench: Benchmarking the Vocal Conversational Abilities for Speech Interaction Models](http://arxiv.org/abs/2505.15727v1), 2025-05-21

(23) [Shared Path: Unraveling Memorization in Multilingual LLMs through Language Similarities](http://arxiv.org/abs/2505.15722v1), 2025-05-21

(24) [Privacy-Preserving Conformal Prediction Under Local Differential Privacy](http://arxiv.org/abs/2505.15721v1), 2025-05-21

(25) [Beyond Empathy: Integrating Diagnostic and Therapeutic Reasoning with Large Language Models for Mental Health Counseling](http://arxiv.org/abs/2505.15715v1), 2025-05-21

(26) [TurnaboutLLM: A Deductive Reasoning Benchmark from Detective Games](http://arxiv.org/abs/2505.15712v1), 2025-05-21

(27) [Advancing LLM Safe Alignment with Safety Representation Ranking](http://arxiv.org/abs/2505.15710v1), 2025-05-21

(28) [LyapLock: Bounded Knowledge Preservation in Sequential Large Language Model Editing](http://arxiv.org/abs/2505.15702v1), 2025-05-21

(29) [HDLxGraph: Bridging Large Language Models and HDL Repositories via HDL Graph Databases](http://arxiv.org/abs/2505.15701v1), 2025-05-21

(30) [Can Large Language Models be Effective Online Opinion Miners?](http://arxiv.org/abs/2505.15695v1), 2025-05-21

(31) [Toward Open Earth Science as Fast and Accessible as Natural Language](http://arxiv.org/abs/2505.15690v1), 2025-05-21

(32) [From Grounding to Manipulation: Case Studies of Foundation Model Integration in Embodied Robotic Systems](http://arxiv.org/abs/2505.15685v1), 2025-05-21

(33) [ThinkLess: A Training-Free Inference-Efficient Method for Reducing Reasoning Redundancy](http://arxiv.org/abs/2505.15684v1), 2025-05-21

(34) [A Federated Splitting Framework for LLMs: Security, Efficiency, and Adaptability](http://arxiv.org/abs/2505.15683v1), 2025-05-21

(35) [UniErase: Unlearning Token as a Universal Erasure Primitive for Language Models](http://arxiv.org/abs/2505.15674v1), 2025-05-21

(36) [Efficient and Direct Duplex Modeling for Speech-to-Speech Language Model](http://arxiv.org/abs/2505.15670v1), 2025-05-21

(37) [Exploring the Limits of Vision-Language-Action Manipulations in Cross-task Generalization](http://arxiv.org/abs/2505.15660v1), 2025-05-21

(38) [Be Careful When Fine-tuning On Open-Source LLMs: Your Fine-tuning Data Could Be Secretly Stolen!](http://arxiv.org/abs/2505.15656v1), 2025-05-21

(39) [Feature Extraction and Steering for Enhanced Chain-of-Thought Reasoning in Language Models](http://arxiv.org/abs/2505.15634v1), 2025-05-21

(40) [Listen to the Context: Towards Faithful Large Language Models for Retrieval Augmented Generation on Climate Questions](http://arxiv.org/abs/2505.15633v1), 2025-05-21

(41) [Can LLMs $\textit{understand}$ Math? -- Exploring the Pitfalls in Mathematical Reasoning](http://arxiv.org/abs/2505.15623v1), 2025-05-21

(42) [DS-Bench: A Realistic Benchmark for Data Science Code Generation](http://arxiv.org/abs/2505.15621v1), 2025-05-21

(43) [LENS: Multi-level Evaluation of Multimodal Reasoning with Large Language Models](http://arxiv.org/abs/2505.15616v1), 2025-05-21

(44) [From Problem-Solving to Teaching Problem-Solving: Aligning LLMs with Pedagogy using Reinforcement Learning](http://arxiv.org/abs/2505.15607v1), 2025-05-21

(45) [Exploring LLM-Generated Feedback for Economics Essays: How Teaching Assistants Evaluate and Envision Its Use](http://arxiv.org/abs/2505.15596v1), 2025-05-21

(46) [Do RAG Systems Suffer From Positional Bias?](http://arxiv.org/abs/2505.15561v1), 2025-05-21

(47) [DayDreamer at CQs-Gen 2025: Generating Critical Questions through Argument Scheme Completion](http://arxiv.org/abs/2505.15554v1), 2025-05-21

(48) [Social Bias in Popular Question-Answering Benchmarks](http://arxiv.org/abs/2505.15553v1), 2025-05-21

(49) [Evaluate Bias without Manual Test Sets: A Concept Representation Perspective for LLMs](http://arxiv.org/abs/2505.15524v1), 2025-05-21

(50) [Protoknowledge Shapes Behaviour of LLMs in Downstream Tasks: Memorization and Generalization with Knowledge Graphs](http://arxiv.org/abs/2505.15501v1), 2025-05-21

(51) [Collaborative Problem-Solving in an Optimization Game](http://arxiv.org/abs/2505.15490v1), 2025-05-21

(52) [KaFT: Knowledge-aware Fine-tuning for Boosting LLMs' Domain-specific Question-Answering Performance](http://arxiv.org/abs/2505.15480v1), 2025-05-21

(53) [LFTF: Locating First and Then Fine-Tuning for Mitigating Gender Bias in Large Language Models](http://arxiv.org/abs/2505.15475v1), 2025-05-21

(54) [PhysicsArena: The First Multimodal Physics Reasoning Benchmark Exploring Variable, Process, and Solution Dimensions](http://arxiv.org/abs/2505.15472v1), 2025-05-21

(55) [CoLA: Collaborative Low-Rank Adaptation](http://arxiv.org/abs/2505.15471v1), 2025-05-21

(56) [A Qualitative Investigation into LLM-Generated Multilingual Code Comments and Automatic Evaluation Metrics](http://arxiv.org/abs/2505.15469v1), 2025-05-21

(57) [Joint Flashback Adaptation for Forgetting-Resistant Instruction Tuning](http://arxiv.org/abs/2505.15467v1), 2025-05-21

(58) [Teaching Language Models to Evolve with Users: Dynamic Profile Modeling for Personalized Alignment](http://arxiv.org/abs/2505.15456v1), 2025-05-21

(59) [ViaRL: Adaptive Temporal Grounding via Visual Iterated Amplification Reinforcement Learning](http://arxiv.org/abs/2505.15447v1), 2025-05-21

(60) [Single LLM, Multiple Roles: A Unified Retrieval-Augmented Generation Framework Using Role-Specific Token Optimization](http://arxiv.org/abs/2505.15444v1), 2025-05-21

(61) [AdUE: Improving uncertainty estimation head for LoRA adapters in LLMs](http://arxiv.org/abs/2505.15443v1), 2025-05-21

(62) [On the Generalization vs Fidelity Paradox in Knowledge Distillation](http://arxiv.org/abs/2505.15442v1), 2025-05-21

(63) [Bridging Sign and Spoken Languages: Pseudo Gloss Generation for Sign Language Translation](http://arxiv.org/abs/2505.15438v1), 2025-05-21

(64) [Set-LLM: A Permutation-Invariant LLM](http://arxiv.org/abs/2505.15433v1), 2025-05-21

(65) [Hunyuan-TurboS: Advancing Large Language Models through Mamba-Transformer Synergy and Adaptive Chain-of-Thought](http://arxiv.org/abs/2505.15431v1), 2025-05-21

(66) [NeoN: A Tool for Automated Detection, Linguistic and LLM-Driven Analysis of Neologisms in Polish](http://arxiv.org/abs/2505.15426v1), 2025-05-21

(67) [Trends and Challenges in Authorship Analysis: A Review of ML, DL, and LLM Approaches](http://arxiv.org/abs/2505.15422v1), 2025-05-21

(68) [Silent Leaks: Implicit Knowledge Extraction Attack on RAG Systems through Benign Queries](http://arxiv.org/abs/2505.15420v1), 2025-05-21

(69) [ClickSight: Interpreting Student Clickstreams to Reveal Insights on Learning Strategies via LLMs](http://arxiv.org/abs/2505.15410v1), 2025-05-21

(70) [Reranking with Compressed Document Representation](http://arxiv.org/abs/2505.15394v1), 2025-05-21

(71) [An Empirical Study of the Anchoring Effect in LLMs: Existence, Mechanism, and Potential Mitigations](http://arxiv.org/abs/2505.15392v1), 2025-05-21

(72) [Are Vision-Language Models Safe in the Wild? A Meme-Based Benchmark Study](http://arxiv.org/abs/2505.15389v1), 2025-05-21

(73) [RePPL: Recalibrating Perplexity by Uncertainty in Semantic Propagation and Language Generation for Explainable QA Hallucination Detection](http://arxiv.org/abs/2505.15386v1), 2025-05-21

(74) [X-WebAgentBench: A Multilingual Interactive Web Benchmark for Evaluating Global Agentic System](http://arxiv.org/abs/2505.15372v1), 2025-05-21

(75) [AI vs. Human Judgment of Content Moderation: LLM-as-a-Judge and Ethics-Based Response Refusals](http://arxiv.org/abs/2505.15365v1), 2025-05-21

(76) [NL-Debugging: Exploiting Natural Language as an Intermediate Representation for Code Debugging](http://arxiv.org/abs/2505.15356v1), 2025-05-21

(77) [FlowKV: Enhancing Multi-Turn Conversational Coherence in LLMs via Isolated Key-Value Cache Management](http://arxiv.org/abs/2505.15347v1), 2025-05-21

(78) [SSR: Speculative Parallel Scaling Reasoning in Test-time](http://arxiv.org/abs/2505.15340v1), 2025-05-21

(79) [Your Language Model Can Secretly Write Like Humans: Contrastive Paraphrase Attacks on LLM-Generated Text Detectors](http://arxiv.org/abs/2505.15337v1), 2025-05-21

(80) [Towards Zero-Shot Differential Morphing Attack Detection with Multimodal Large Language Models](http://arxiv.org/abs/2505.15332v1), 2025-05-21

(81) [Improving LLM First-Token Predictions in Multiple-Choice Question Answering via Prefilling Attack](http://arxiv.org/abs/2505.15323v1), 2025-05-21

(82) [Emotional Supporters often Use Multiple Strategies in a Single Turn](http://arxiv.org/abs/2505.15316v1), 2025-05-21

(83) [Trajectory Bellman Residual Minimization: A Simple Value-Based Method for LLM Reasoning](http://arxiv.org/abs/2505.15311v1), 2025-05-21

(84) [Multiple Weaks Win Single Strong: Large Language Models Ensemble Weak Reinforcement Learning Agents into a Supreme One](http://arxiv.org/abs/2505.15306v1), 2025-05-21

(85) [Chinese Toxic Language Mitigation via Sentiment Polarity Consistent Rewrites](http://arxiv.org/abs/2505.15297v1), 2025-05-21

(86) [LLM-Explorer: A Plug-in Reinforcement Learning Policy Exploration Enhancement Driven by Large Language Models](http://arxiv.org/abs/2505.15293v1), 2025-05-21

(87) [Hallucinate at the Last in Long Response Generation: A Case Study on Long Document Summarization](http://arxiv.org/abs/2505.15291v1), 2025-05-21

(88) [Web-Shepherd: Advancing PRMs for Reinforcing Web Agents](http://arxiv.org/abs/2505.15277v1), 2025-05-21

(89) [Scaling Diffusion Transformers Efficiently via $μ$P](http://arxiv.org/abs/2505.15270v1), 2025-05-21

(90) [LiveVLM: Efficient Online Video Understanding via Streaming-Oriented KV Cache and Retrieval](http://arxiv.org/abs/2505.15269v1), 2025-05-21

(91) [Blind Spot Navigation: Evolutionary Discovery of Sensitive Semantic Concepts for LVLMs](http://arxiv.org/abs/2505.15265v1), 2025-05-21

(92) [ReGUIDE: Data Efficient GUI Grounding via Spatial Reasoning and Search](http://arxiv.org/abs/2505.15259v1), 2025-05-21

(93) [When Less Language is More: Language-Reasoning Disentanglement Makes LLMs Better Multilingual Reasoners](http://arxiv.org/abs/2505.15257v1), 2025-05-21

(94) [MentalMAC: Enhancing Large Language Models for Detecting Mental Manipulation via Multi-Task Anti-Curriculum Distillation](http://arxiv.org/abs/2505.15255v1), 2025-05-21

(95) [Towards Explainable Temporal Reasoning in Large Language Models: A Structure-Aware Generative Framework](http://arxiv.org/abs/2505.15245v1), 2025-05-21

(96) [Adaptive Plan-Execute Framework for Smart Contract Security Auditing](http://arxiv.org/abs/2505.15242v1), 2025-05-21

(97) [Generalised Probabilistic Modelling and Improved Uncertainty Estimation in Comparative LLM-as-a-judge](http://arxiv.org/abs/2505.15240v1), 2025-05-21

(98) [Multilingual Prompting for Improving LLM Generation Diversity](http://arxiv.org/abs/2505.15229v1), 2025-05-21

(99) [R-TOFU: Unlearning in Large Reasoning Models](http://arxiv.org/abs/2505.15214v1), 2025-05-21

(100) [Deliberation on Priors: Trustworthy Reasoning of Large Language Models on Knowledge Graphs](http://arxiv.org/abs/2505.15210v1), 2025-05-21
